server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -251,6 +251,7 @@ public Config getConfig() {
   public TabletServerResourceManager(ServerContext context, TabletHostingServer tserver) {
     this.context = context;
     final AccumuloConfiguration acuConf = context.getConfiguration();
+    final boolean enableMetrics = acuConf.getBoolean(Property.GENERAL_MICROMETER_ENABLED);
```

Comment:
Can you use context.getMetricsInfo().metricsEnabled() ?

Reply:
Fixed in 601f167e80